<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TinySyntaxTree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="tinyFileEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>481</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="tinyFileEditLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tiny源程序</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>460</width>
      <height>561</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="word">
     <attribute name="title">
      <string>词法分析</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>453</width>
        <height>525</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>453</width>
        <height>525</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>453</width>
        <height>525</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="syntax">
     <attribute name="title">
      <string>语法分析</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>452</width>
        <height>531</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>452</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>452</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="errLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>存在语法错误</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="tinyFileEditLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>错误列表</string>
    </property>
   </widget>
   <widget class="QTableView" name="errTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>961</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
   <addaction name="actionsave"/>
   <addaction name="actionanalyse"/>
  </widget>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="../LALR/resource.qrc">
     <normaloff>:/icon/open.png</normaloff>:/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开源程序</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="../LALR/resource.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存源程序</string>
   </property>
  </action>
  <action name="actionanalyse">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/analyse.png</normaloff>:/icon/analyse.png</iconset>
   </property>
   <property name="text">
    <string>分析源程序</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
